LEAF_Request_Portal/sources/TemplateFileHistory.php

```diff
@@ -0,0 +1,145 @@
+<?php
```

Comment:
This entire file should be gone through with the psr 12 coding standards

Reply:
I have made the changes